### PR TITLE
Updating Slicer to integrate the changes of SEM

### PR DESCRIPTION
**Pull request by jbvimort**
This pull request is following one made on Slicer Execution Model: https://github.com/Slicer/SlicerExecutionModel/pull/91

It is doing 3 distinct things:
- Updating the SEM's version used in Slicer
- Updating Slicer's code in order to use the newly created functions in SEM instead of the deprecated ones
- Updating and depreciating one function of Slicer in order to have a more explicit name (GetParameterValue instead of GetParameterDefault)

### CODE REVIEW

File: Libs/MRML/CLI/vtkMRMLCommandLineModuleNode.h [hunk 1]
```diff
@@ -273,7 +273,9 @@ class VTK_MRML_CLI_EXPORT vtkMRMLCommandLineModuleNode : public vtkMRMLNode
   std::string GetParameterDescription(unsigned int group, unsigned int param) const;
   std::string GetParameterChannel(unsigned int group, unsigned int param) const;
   std::string GetParameterIndex(unsigned int group, unsigned int param) const;
+  //GetParameterDefault is depreciated you should think about using GetParameterValue
```
Comment (vovythevov) [inline Libs/MRML/CLI/vtkMRMLCommandLineModuleNode.h]: You should use the tag "\deprecated" here.

**Comment by pieper** [comment]
Hi Jb - Probably makes sense to squash the commits for clarity and also include a link to the discussion.  Also a spell checker on the commit messages could help 😄

**Comment by jbvimort** [comment]
Should I squash the 3 commits into one?

**Comment by vovythevov** [comment]
@jbvimort: I think that makes sense to squash all of them together.

**Comment by jbvimort** [comment]
I hope it's looking better now! :smile:

**Comment by pieper** [comment]
👍

**Comment by jcfr** [comment]
Thanks @jbvimort for working on this and thanks @vovythevov and @pieper for the reviews.

Integrated in [r26137](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=26137)

**Pull request closed without merging**